diff --git a/node_modules/react-dom/cjs/react-dom-client.development.js b/node_modules/react-dom/cjs/react-dom-client.development.js
index dd10829..f56b624 100644
--- a/node_modules/react-dom/cjs/react-dom-client.development.js
+++ b/node_modules/react-dom/cjs/react-dom-client.development.js
@@ -5610,10 +5610,6 @@
         didScheduleRenderPhaseUpdateDuringThisPass && (thenableState$1 = null);
         thenableIndexCounter$1 = 0;
         didScheduleRenderPhaseUpdateDuringThisPass = !1;
-        if (numberOfReRenders >= RE_RENDER_LIMIT)
-          throw Error(
-            "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-          );
         numberOfReRenders += 1;
         ignorePreviousDependencies = !1;
         workInProgressHook = currentHook = null;
diff --git a/node_modules/react-dom/cjs/react-dom-profiling.development.js b/node_modules/react-dom/cjs/react-dom-profiling.development.js
index b8842ab..8179444 100644
--- a/node_modules/react-dom/cjs/react-dom-profiling.development.js
+++ b/node_modules/react-dom/cjs/react-dom-profiling.development.js
@@ -5618,10 +5618,6 @@
         didScheduleRenderPhaseUpdateDuringThisPass && (thenableState$1 = null);
         thenableIndexCounter$1 = 0;
         didScheduleRenderPhaseUpdateDuringThisPass = !1;
-        if (numberOfReRenders >= RE_RENDER_LIMIT)
-          throw Error(
-            "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-          );
         numberOfReRenders += 1;
         ignorePreviousDependencies = !1;
         workInProgressHook = currentHook = null;
diff --git a/node_modules/react-dom/cjs/react-dom-server-legacy.browser.development.js b/node_modules/react-dom/cjs/react-dom-server-legacy.browser.development.js
index 7b5c759..3e7f862 100644
--- a/node_modules/react-dom/cjs/react-dom-server-legacy.browser.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server-legacy.browser.development.js
@@ -3865,10 +3865,6 @@
       return nextCreate;
     }
     function dispatchAction(componentIdentity, queue, action) {
-      if (25 <= numberOfReRenders)
-        throw Error(
-          "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-        );
       if (componentIdentity === currentlyRenderingComponent)
         if (
           ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js b/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js
index 004f577..8390ed5 100644
--- a/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server-legacy.node.development.js
@@ -3865,10 +3865,6 @@
       return nextCreate;
     }
     function dispatchAction(componentIdentity, queue, action) {
-      if (25 <= numberOfReRenders)
-        throw Error(
-          "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-        );
       if (componentIdentity === currentlyRenderingComponent)
         if (
           ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.js b/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.js
index fb5eaa2..605c1ef 100644
--- a/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server-legacy.node.production.js
@@ -3161,10 +3161,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders)
-    throw Error(
-      "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-    );
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.browser.development.js b/node_modules/react-dom/cjs/react-dom-server.browser.development.js
index 93df649..edb233b 100644
--- a/node_modules/react-dom/cjs/react-dom-server.browser.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server.browser.development.js
@@ -4004,10 +4004,6 @@
       return nextCreate;
     }
     function dispatchAction(componentIdentity, queue, action) {
-      if (25 <= numberOfReRenders)
-        throw Error(
-          "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-        );
       if (componentIdentity === currentlyRenderingComponent)
         if (
           ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.bun.development.js b/node_modules/react-dom/cjs/react-dom-server.bun.development.js
index b3491f1..bd6b2bb 100644
--- a/node_modules/react-dom/cjs/react-dom-server.bun.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server.bun.development.js
@@ -4997,10 +4997,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders)
-    throw Error(
-      "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-    );
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.bun.production.js b/node_modules/react-dom/cjs/react-dom-server.bun.production.js
index 5b7d588..5691d34 100644
--- a/node_modules/react-dom/cjs/react-dom-server.bun.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server.bun.production.js
@@ -3130,10 +3130,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders)
-    throw Error(
-      "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-    );
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.edge.development.js b/node_modules/react-dom/cjs/react-dom-server.edge.development.js
index 2cf0f85..38f5bfb 100644
--- a/node_modules/react-dom/cjs/react-dom-server.edge.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server.edge.development.js
@@ -4000,10 +4000,6 @@
       return nextCreate;
     }
     function dispatchAction(componentIdentity, queue, action) {
-      if (25 <= numberOfReRenders)
-        throw Error(
-          "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-        );
       if (componentIdentity === currentlyRenderingComponent)
         if (
           ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.edge.production.js b/node_modules/react-dom/cjs/react-dom-server.edge.production.js
index de09c32..98f5dba 100644
--- a/node_modules/react-dom/cjs/react-dom-server.edge.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server.edge.production.js
@@ -3512,10 +3512,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders)
-    throw Error(
-      "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-    );
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.node.development.js b/node_modules/react-dom/cjs/react-dom-server.node.development.js
index 87c435a..b52de0e 100644
--- a/node_modules/react-dom/cjs/react-dom-server.node.development.js
+++ b/node_modules/react-dom/cjs/react-dom-server.node.development.js
@@ -3906,10 +3906,6 @@
       return nextCreate;
     }
     function dispatchAction(componentIdentity, queue, action) {
-      if (25 <= numberOfReRenders)
-        throw Error(
-          "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-        );
       if (componentIdentity === currentlyRenderingComponent)
         if (
           ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.node.production.js b/node_modules/react-dom/cjs/react-dom-server.node.production.js
index 0b983ee..b7a1ab1 100644
--- a/node_modules/react-dom/cjs/react-dom-server.node.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server.node.production.js
@@ -3415,10 +3415,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders)
-    throw Error(
-      "Too many re-renders. React limits the number of renders to prevent an infinite loop."
-    );
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-client.production.js b/node_modules/react-dom/cjs/react-dom-client.production.js
index 2af1b36..9803393 100644
--- a/node_modules/react-dom/cjs/react-dom-client.production.js
+++ b/node_modules/react-dom/cjs/react-dom-client.production.js
@@ -3534,7 +3534,6 @@ function renderWithHooksAgain(workInProgress, Component, props, secondArg) {
     didScheduleRenderPhaseUpdateDuringThisPass && (thenableState$1 = null);
     thenableIndexCounter$1 = 0;
     didScheduleRenderPhaseUpdateDuringThisPass = !1;
-    if (25 <= numberOfReRenders) throw Error(formatProdErrorMessage(301));
     numberOfReRenders += 1;
     workInProgressHook = currentHook = null;
     if (null != workInProgress.updateQueue) {
diff --git a/node_modules/react-dom/cjs/react-dom-profiling.profiling.js b/node_modules/react-dom/cjs/react-dom-profiling.profiling.js
index 931305c..24dcfa2 100644
--- a/node_modules/react-dom/cjs/react-dom-profiling.profiling.js
+++ b/node_modules/react-dom/cjs/react-dom-profiling.profiling.js
@@ -3682,7 +3682,6 @@ function renderWithHooksAgain(workInProgress, Component, props, secondArg) {
     didScheduleRenderPhaseUpdateDuringThisPass && (thenableState$1 = null);
     thenableIndexCounter$1 = 0;
     didScheduleRenderPhaseUpdateDuringThisPass = !1;
-    if (25 <= numberOfReRenders) throw Error(formatProdErrorMessage(301));
     numberOfReRenders += 1;
     workInProgressHook = currentHook = null;
     if (null != workInProgress.updateQueue) {
diff --git a/node_modules/react-dom/cjs/react-dom-server-legacy.browser.production.js b/node_modules/react-dom/cjs/react-dom-server-legacy.browser.production.js
index 5fe3e08..c1c8864 100644
--- a/node_modules/react-dom/cjs/react-dom-server-legacy.browser.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server-legacy.browser.production.js
@@ -3122,7 +3122,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders) throw Error(formatProdErrorMessage(301));
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
diff --git a/node_modules/react-dom/cjs/react-dom-server.browser.production.js b/node_modules/react-dom/cjs/react-dom-server.browser.production.js
index c4e2710..e94ee05 100644
--- a/node_modules/react-dom/cjs/react-dom-server.browser.production.js
+++ b/node_modules/react-dom/cjs/react-dom-server.browser.production.js
@@ -3481,7 +3481,6 @@ function useMemo(nextCreate, deps) {
   return nextCreate;
 }
 function dispatchAction(componentIdentity, queue, action) {
-  if (25 <= numberOfReRenders) throw Error(formatProdErrorMessage(301));
   if (componentIdentity === currentlyRenderingComponent)
     if (
       ((didScheduleRenderPhaseUpdate = !0),
